packages/template-retail-react-app/app/utils/test-utils.js

```diff
@@ -99,7 +100,8 @@ export const TestProviders = ({
     appConfig = mockConfig.app,
     siteAlias = DEFAULT_SITE,
     isGuest = false,
-    bypassAuth = true
+    bypassAuth = true,
+    basket = {bonusDiscountLineItems: []}
```

Comment:
IMO, TestProviders should be agnostic of the storefront data, and be as generic as possible.